pkg/services/apps/v2.go

```diff
@@ -40,7 +40,15 @@ func (svc *V2Service) Query(query *AppsQuery) ([]App, error) {
 	}
 
 	var apps []App
-	json.Unmarshal(resp, &apps)
+	for _, bytes := range resp {
+		var unmarshalled []App
+		json.Unmarshal(bytes, &unmarshalled)
+		if len(apps) == 0 {
+			apps = unmarshalled
+		} else {
+			apps = append(apps, unmarshalled...)
```

Comment:
You will see this pattern in multiple v2's: It is related to tests requiring an empty array when the query is valid but there are no matches found.